data/lm/README.rst

```diff
@@ -8,18 +8,27 @@ lm.binary was generated from the LibriSpeech normalized LM training text, availa
    import os
 
    from urllib import request
+   from collections import Counter
 
    # Grab corpus.
    url = 'http://www.openslr.org/resources/11/librispeech-lm-norm.txt.gz'
    data_upper = '/tmp/upper.txt.gz'
```

Comment:
I know this is no part of your commit, but should we make this platform independent with something like `tempfile`?